README.md

```diff
@@ -21,6 +21,172 @@ def deps do
 end
 ```
 
+## Use cases
```

Comment:
Pluggy has something about this in the **Application** session in their [site](https://pluggy.ai/en)

![image](https://user-images.githubusercontent.com/1009242/116104326-bb1cfe00-a686-11eb-9988-7f0ab89dfc02.png)

Reply:
Yes! We can remove use cases. As I said was a PR idea to be validated, but the problem is how to show the liber's power quickly, for example, when a developer is navigating through Hexdocs or Github. When you see a snippet of code working and examples of use, you have more engagement